README.md

```diff
@@ -30,7 +35,17 @@ Your Answer:
 
 ```js
 // your code here
+function Instructor(name){
+
+this.name = name
+givesHomework = function(assignment){
+  console.log(name , assignment);
+}  
+}
+
+Instructor.prototype.Robin = function givesHomework(Intro to Ruby){
 
```

Comment:
to create a new instance of an `instructor` and call the method on it, looks like:
```js
var robin = new Instructor("Robin")
robin.givesHomework("Intro to Ruby")
```